src/NuGet.Services.Logging/project.json

```diff
@@ -1,8 +1,8 @@
 {
     "version": "2.0.0-rc2-3",

```

Comment:
Does this also need to be updated in the AssmeblyInfo.cs?

---
In reply to: [74493268](https://github.com/NuGet/NuGet.Staging/pull/71#discussion_r74493268) [](ancestors = 74493268)